pallets/creditcoin/src/tests.rs

```diff
@@ -3082,13 +3107,13 @@ fn register_address_v2_should_error_when_address_too_long() {
 		let proof = OwnershipProof::EthSign(ownership_proof);
 		assert_noop!(
 			Creditcoin::register_address_v2(Origin::signed(who), blockchain, address, proof),
-			crate::Error::<Test>::EthSignExternalAddressGenerationFailed
+			crate::Error::<Test>::MalformedExternalAddress
 		);
 	})
 }
 
 #[test]
-fn register_address_v2_should_error_when_signature_is_invalid() {
+fn try_extract_address_should_error_when_signature_is_invalid() {
```

Comment:
Wrt my previous comment that the test target here is `try_extract_address()` I think I didn't explain clearly.

If you are going to change the function name so that it will be executing tests against `try_extract_address()` then you need to change the test function body as well. You can call `try_extract_address()` directly and you don't need to go all the way through calling an extrinsic in order to exercise `try_extract_address()`.

I think you should revert the name change and then adjust this test func so that it tests `register_address_v2` but does so properly.

See below for more context.